### ISSUE TITLE
[Accessibility][Usability]A11y_VSCode_ViewSearch_Keyboard: Keyboard focus is not clearly visible on checkboxes when already checked. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52605]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
Issue Type : Usability

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Search"(Cntrl+Shift+F) button.
3. Navigate to checkboxes - "Match Case" , "Match Whole Word" , "Use RegEx", "Use Exclude settings and Ignore Files" and check them.

**Actual:**
When we navigate from "Search" text field to move to next controls i.e. checkboxes the focus is not visible when all the checkboxes are checked. Unable to identify the currently focused checkbox. 

Same issue is for "Use Exclude settings and Ignore Files" checkbox where user is unable to identify focus when checkbox is already checked. 

**Expected:**
Focus on "Match Case" , "Match Whole Word" , "Use RegEx", "Use Exclude settings and Ignore Files" (in checked state) checkboxes should be clearly visible when focused.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Low vision /visually impaired users will not be able to identify the focus on focused element when all the checkboxes are checked.  

**Attachment For Reference:**
[A11y_VSCode_ViewSearch_Usability_FocusIssueOnCheckboxes.pptx](https://github.com/Microsoft/vscode/files/2123617/A11y_VSCode_ViewSearch_Usability_FocusIssueOnCheckboxes.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
Basically a theme issue, we also have https://github.com/Microsoft/vscode/issues/48855 for this. Also #8017.

For seeing-impaired users, they may want to use the "high contrast" theme.
